Bottom: fffaae4f1f4a131a0c4b9c1faec6853a2076a831
Top:    c359b83491356f0276b7e28f980fa801e93602e6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-03 22:05:59 +0000

Refresh of bentley-ottann-test-implementa

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 8f57701..1d1db7b 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -37,7 +37,7 @@ PCB_SRCS = \
 	box.h \
 	buffer.c \
 	buffer.h \
-	cairo.cairo.h \
+	cairo/cairo.h \
 	cairo/cairoint.h \
 	cairo/cairo-atomic-private.h \
 	cairo/cairo-bentley-ottmann.c \
